### ISSUE TITLE
CNN Hyperparameter tuning

### ISSUE THREAD

**Issue by Cyrof** [Cyrof/Artificial-Intelligence-Autonomous-Systems#9]
# Hyperparameter tuning required

The CNN model is more or less completed. But it still requires some hyperparameter tuning to slightly increase the accuracy of the model.

### Hyperparameter to use
- Number of layers (use `module_list` from pytorch)
- Stride
- Padding
- Learning rate (use dynamic)
- Batch size (use dynamic)

**Comment by Cyrof** [comment]
# Important Notes 
Can delete CLI prompt for batch size, epoch, and learning rate to make it dynamic. These feature tuning methods can be made dynamic.
